stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -135,6 +135,16 @@ spec:
           mountPath: /opt/bitnami/redis/etc/redis.conf
           subPath: redis.conf
         {{- end }}
+      {{- if and .Values.master.persistence.enabled (not .Values.persistence.existingClaim) | and .Values.master.securityContext.enabled }}
```

Comment:
Why do you need the pipe?

Reply:
i am no so familiar with go template
reworked